recipes/meson-python/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "meson-python" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/meson_python-{{ version }}.tar.gz
+  sha256: 841d68a9ff26b7f75c8aab23008966abc52646846e6f1b50de55f8961834ed0f
+  patches:
+    - ninja-overdep.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - colorama
+    - meson >=0.60.0
+    - ninja
```

Comment:
I think `wheel` should be here; must have overlooked that before. It gets pulled in anyway via `python` I guess, but it's a direct dependency so good to capture.